proto/proto/ibc.proto

```diff
@@ -38,6 +39,31 @@ message IBCAction {
   }
 }
 
+message ICS20Withdrawal {
+  // the chain ID of the destination chain for this ICS20 transfer
+  string destination_chain_id = 1;
+
+  // a transparent value consisting of an amount and an asset ID.
+  crypto.Value value = 2;
+
+  // the address on the destination chain to send the transfer to 
+  string destination_chain_address = 3;
+
+  // a "sender" penumbra address to use to return funds from this withdrawal.
+  // this should be an ephemeral address
+  crypto.Address sender = 5;
```

Comment:
Since this has a different meaning than normal, we should rename it, perhaps to `return_address`, since that's what it's used for?